src/test/nft.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity >=0.5.12;
+
+import "ds-test/test.sol";
+
+import { ERC721  } from "../openzeppelin-solidity/token/ERC721/ERC721.sol";
+
+// This contract allows anyone to mint an NFT. Used for testing Tinlake
+contract SimpleNFT is ERC721 {
```

Comment:
I don't think the SimpleNFT belongs into the title repo. Why would you put it here?

Reply:
To add some more context: it's not needed by anything in the title why would it go in here then?

Reply:
Because keeping it in tinlake-core requires that we need to keep the OpenZeppelin-solidity directory there as well just for the system test mock. Or, we just directly use ERC721 instead of SimpleNFT in tinlake -- I feel like it is overkill to have the whole OZ directory for just that one case. wdyt?

Reply:
I am going to remove SimpleNFT in tinlake and just use ERC721